IfcElectricAppliance Rail Import BIMQ 2021-07-21T06:47:54.2997685Z (#160)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/t/TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK/Documentation.md

> Indicate the stuck rate of tickets in the turnstile.

Author: ```suggestion
Indicate the frequency of tickets being stuck or jammed in appliance..
```

Reviewer: Updated, but changed "frequency" to "ratio". https://github.com/bSI-InfraRoom/IFC-Specification/pull/160/commits/8b12068ba20d88a43f933b941b15b54fce5e5c4d

> Indicates the ratio of tickets being stuck or jammed in appliance or turnstile.

Reviewer 2: Was this property added somewhere already? Don't see it in this PR.
```suggestion
Indicates the ratio of tickets being stuck or jammed in the appliance or turnstile.
```

Reviewer 3: Yes it was in IFC door for turnstiles
